src/Microsoft.AspNet.Server.Kestrel/project.json

```diff
@@ -52,13 +52,13 @@
     "keyFile": "../../tools/Key.snk"
   },
   "scripts": {
-    "prepare": [
-      "dnu restore ../../tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode",
-      "dnx -p ../../tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode run Http/FrameHeaders.Generated.cs Http/Frame.Generated.cs"
+    "prepublish": [
+      "dotnet restore ../../tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode",
+      "dotnet run -p ../../tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode Http/FrameHeaders.Generated.cs Http/Frame.Generated.cs"
     ],
-    "postrestore": [
-      "dnu restore ../../tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier",
-      "dnx -p ../../tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier run"
+    "precompile": [
```

Comment:
I have to take this out of here because it gets runs over and over. I'm thinking of moving this step to `build.cmd`/`build.sh` until we have `postrestore` in nuget. Any suggestions for alternatives?